.github/workflows/merge-stable.yml

```diff
@@ -0,0 +1,31 @@
+name: Stable branch auto merge
+
+on:
+  push:
+    branches: [ stable ]
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+    steps:
+      # First: merge
+      - uses: octokit/request-action@v2.x
+        id: merge_action
+        with:
+          route: POST /repos/:repository/merges
+          repository: ${{ github.repository }}
+          base: master
+          head: stable
+        env:
+          GITHUB_TOKEN: ${{ secrets.STABLE_MERGE_GITHUB_TOKEN }}
+      # Report errors if any
+      - uses: rtCamp/action-slack-notify@v2.0.2
+        id: error_message_slack
+        name: Slack notification
+        if: (steps.merge_action.outputs.status != 201) && (steps.merge_action.outputs.status != 204)
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_CHANNEL: "cke5-code"
```

Comment:
Could we get this to cke5-ci? I think that cke5-code is mostly dead.

Reply:
Wanted to bring it for a discussion.

I made it `cke5-code` temporarily. For me `cke5-ci` sounds best, good visibility and not creating a dedicated channel for something that should happen _extremely rarely_.

Reply:
Wanted to bring it for a discussion.

I made it `cke5-code` temporarily. For me `cke5-ci` sounds best, good visibility and not creating a dedicated channel for something that should happen _extremely rarely_.